app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -41,6 +41,11 @@ def confirm
       )
       properties = result.to_h.merge(analytics_properties)
       analytics.multi_factor_auth_setup(**properties)
+
+      if @platform_authenticator
+        irs_attempts_api_tracker.mfa_enroll_webauthn_platform(success: result.success?)
+      end
```

Comment:
Are you handling the non-platform event in a separate PR?

Reply:
Yessir - separate event, separate story, separate PR...